Automattic/jetpack
# E2E tests: run against arbitrary site

#### Changes proposed in this Pull Request:

* Add support for running the e2e tests against any test site.

#### Jetpack product discussion
p9dueE-3dC-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* E2E tests in CI (unchanged, running against a Docker setup) should pass
* Using a test site: 
    + Add your test site configuration in the local config file and run the relevant tests with `TEST_SITE=site_name_config_property`. Only some tests make sense to un, depending on your site configuration.
or
    + Use the existing test site config, and run the tests using `TEST_SITE=test`
